Platform/CommonBoardPkg/Tools/GpioDataConvert.py

```diff
@@ -107,13 +109,21 @@
 
 GPIO_MIXED_INT_CONFIG = {
     'GpioIntLevel|GpioIntApic'  : 0x31,
+    'GpioIntApic|GpioIntLevel'  : 0x31,
     'GpioIntLevel|GpioIntSci'   : 0x29,
+    'GpioIntSci|GpioIntLevel'   : 0x29,
     'GpioIntLevel|GpioIntSmi'   : 0x25,
+    'GpioIntSmi|GpioIntLevel'   : 0x25,
     'GpioIntEdge|GpioIntSci'    : 0x69,
+    'GpioIntSci|GpioIntEdge'    : 0x69,
     'GpioIntEdge|GpioIntApic'   : 0x71,
+    'GpioIntApic|GpioIntEdge'   : 0x71,
     'GpioIntEdge|GpioIntSmi'    : 0x65,
+    'GpioIntSmi|GpioIntEdge'    : 0x65,
     'GpioIntEdge|GpioIntDefault': 0x60,
+    'GpioIntDefault|GpioIntEdge': 0x60,
     'GpioIntLvlEdgDis|GpioIntDis':0xA1,
+    'GpioIntDis|GpioIntLvlEdgDis':0xA1,

```

Comment:
Better to put each key and value separately in this dictionary. A caller needs to do OR according to input.
ex)
```
GPIO_INIT_CONFIG = {
GpioIntDefault : 0x00,
GpioIntDis : 0x01,
...
GpioIntLvlEdgDis : 0xA0,
...
}
```

Reply:
GPIO_INT_CONFIG is already provided. But some csv/h files may use this mixed config ,so added this new dict so that I can directly use the ORed config key to get its value.

Reply:
I mean we can use GPIO_INT_CONFIG for mixed one as well.
```
    if '|' not in parts[5]:
        sbl_dw0.Dw0Tmpl.InterruptConfig = GPIO_INT_CONFIG[parts[5]]
    else:
        sbl_dw0.Dw0Tmpl.InterruptConfig = GPIO_MIXED_INT_CONFIG[parts[5]]
```
->
```
for key in parts[5].split(':'):
  sbl_dw0.Dw0Tmpl.InterruptConfig |= GPIO_INT_CONFIG[key]
```

Reply:
typo. split('|')